core/analytics-enabled/src/main/kotlin/com/wire/android/feature/analytics/AnonymousAnalyticsRecorderImpl.kt

```diff
@@ -42,8 +43,12 @@ class AnonymousAnalyticsRecorderImpl : AnonymousAnalyticsRecorder {
         )
             .enableTemporaryDeviceIdMode() // Nothing is sent until a proper ID is placed
             .setLoggingEnabled(analyticsSettings.enableDebugLogging)
+        countlyConfig.apm.enableAppStartTimeTracking()
+        countlyConfig.apm.enableForegroundBackgroundTracking()
+        countlyConfig.setApplication(context.applicationContext as Application)
 
         Countly.sharedInstance().init(countlyConfig)
+        Countly.sharedInstance().consent().giveConsent(arrayOf("apm"))
```

Comment:
Runtime consent given to the SDK for performance metrics, only enabled if analytics enabled.